java/src/main/native/src/CudaJni.cpp

```diff
@@ -216,6 +216,21 @@ JNIEXPORT jint JNICALL Java_ai_rapids_cudf_Cuda_getNativeComputeMode(JNIEnv* env
   CATCH_STD(env, -2);
 }
 
+JNIEXPORT jbyteArray JNICALL Java_ai_rapids_cudf_Cuda_getNativeGpuUuid(JNIEnv* env, jclass)
+{
+  try {
+    cudf::jni::auto_set_device(env);
+    int device;
+    CUDF_CUDA_TRY(cudaGetDevice(&device));
+    cudaDeviceProp device_prop;
+    CUDF_CUDA_TRY(cudaGetDeviceProperties(&device_prop, device));
```

Comment:
I am not seeing this in cuda13 breaking changes, but please still help double-check if this code works for both cuda12 and 13, thanks 

ref: https://github.com/rapidsai/cudf/pull/19645/files

Reply:
Building on cuda 13 passed.